crates/rerun/src/run.rs

```diff
@@ -263,7 +263,14 @@ async fn run_impl(
     let rx = if let Some(url_or_path) = args.url_or_path.clone() {
         match categorize_argument(url_or_path) {
             ArgumentCategory::RrdHttpUrl(url) => {
-                re_viewer::stream_rrd_from_http::stream_rrd_from_http_to_channel(url)
+                #[cfg(feature = "native_viewer")]
+                {
+                    re_viewer::stream_rrd_from_http::stream_rrd_from_http_to_channel(url)
+                }
+                #[cfg(not(feature = "native_viewer"))]
+                {
+                    anyhow::bail!("Can't start viewer - rerun was compiled without the 'native_viewer' feature");
```

Comment:
I think this is the wrong fix - streaming rrd files from http has nothing to do with the native viewer - we should instead move `stream_rrd_from_http_to_channel` to some other crate imho

Reply:
I'll make the fix in https://github.com/rerun-io/rerun/pull/1772